x-pack/legacy/plugins/siem/server/lib/detection_engine/scripts/rules/queries/query_with_errors.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Query with a rule id",
+  "description": "Query with a rule_id that acts like an external id",
+  "rule_id": "query-rule-id-errors-2",
+  "risk_score": 1,
+  "interval": "30s",
+  "severity": "high",
+  "type": "query",
+  "from": "now-6m",
+  "to": "now",
+  "query": "user.name: root devin user.name: admin"
+}
```

Comment:
Is it possible to really quickly move this to `./rules/test_cases`. I try to keep the corner cases and errors in there so people don't accidentally use this an example